fastchat/serve/monitor/monitor.py

```diff
@@ -856,6 +857,119 @@ def build_category_leaderboard_tab(
     )
 
 
+def compute_ub_ranking(arena_df):
```

Comment:
I'm surprised there's no function that already exists for this?

Reply:
Updated so uses recompute_final_ranking instead.